src/components/shared/Navbar/navbar.styles.js

```diff
@@ -123,10 +126,12 @@ export const Logo = styled(RawLogo)`
 export const SignInButton = styled(Button)`
   background-color: transparent;
   font-size: 0.9rem;
```

Comment:
Please adhere to css styling best practices
```suggestion
  font-size: 1rem;
```

Reply:
QA want it like this